### PR TITLE
Day 7 Slug

### PR DESCRIPTION
**Pull request by Noorieee**
{A14795C7-7D9A-48F9-B9A2-5CE4D432BCD7}

### CODE REVIEW

File: src/components/Slug/Slug.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  font-weight: 400;
+`
+
+const Slug = (): JSX.Element => {
+  const [value, setValue] = useState('')
+  
+  const slug = value.trim().length > 0 ? `/${value.trim().split(' ').join('-').toLowerCase()}` : '';
+  
+  const handleValueChange = (e:any):void => {
+    console.log('e', e.target.value)
```
Comment (goralight) [inline src/components/Slug/Slug.tsx]: remove logs

File: src/components/Slug/Slug.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  line-height: 19px;
+  font-weight: 400;
+`
+
+const Slug = (): JSX.Element => {
+  const [value, setValue] = useState('')
+  
+  const slug = value.trim().length > 0 ? `/${value.trim().split(' ').join('-').toLowerCase()}` : '';
+  
+  const handleValueChange = (e:any):void => {
```
Comment (goralight) [inline src/components/Slug/Slug.tsx]: shouldnt be using `any` type

Comment (goralight) [reply src/components/Slug/Slug.tsx]: also no space between : and void

File: src/components/Slug/Slug.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    console.log('e', e.target.value)
+    console.log('Input Value:', e.target.value);
+    setValue(e.target.value)
+  }
+
+  return (
+    <Container>
+      <InputContainer>
+        <Label htmlFor="titleInput">Title</Label>
```
Comment (goralight) [inline src/components/Slug/Slug.tsx]: dont use double quotes use single 🦦

File: src/components/Slug/Slug.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        <StyledInput value={value} onChange={handleValueChange}/> 
+      </InputContainer>
+      {slug && (
+        <div>
+          <SlugText>{slug}</SlugText>
+        </div>
+      )}
+    </Container>
+  )
+
```
Comment (goralight) [inline src/components/Slug/Slug.tsx]: remove extra line

File: src/components/Slug/Slug.tsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        <div>
+          <SlugText>{slug}</SlugText>
+        </div>
+      )}
+    </Container>
+  )
+  
+}
+  
+export default Slug
```
Comment (goralight) [inline src/components/Slug/Slug.tsx:72]: no extra line at end of file

File: src/components/Slug/Slug.tsx [hunk 6]
```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+
+const Container = styled.div`
+  background-color: #ffffff;
+  width: 100%;
+  padding: 30px 46px;
+`
+const InputContainer = styled.div`
```
Comment (goralight) [inline src/components/Slug/Slug.tsx:8]: check these stylings to make sure there are none which you are settings which are already the default values, eg width: auto / text-align: left, etc.

File: src/components/Slug/Slug.tsx (hunk 5 above)
Comment (Noorieee) [reply src/components/Slug/Slug.tsx:72]: There is a line on my vscode but I'm not sure why it's not showing up on here

**Review by goralight — APPROVED** [review]

**Pull request merged**